thermod/runBARCO.R

```diff
@@ -172,3 +131,53 @@ g4 <- ggplot(output) +
   labs(x = 'Simulated Time', y = 'Fluxes in mg/(cm3 d)')  +
   theme_bw()+
   theme(legend.position="bottom");g4
+
+
+
+### forecasting framework
+
+for (ii in 1:3){
+  print(paste0('run number: ', ii))
+
+  boundary_noise <- add_noise(boundary)
+
+  for (jj in 1:100){
+    y_noise <- yini
+    y_noise[1] <- rnorm(1, mean = y_noise[1], sd = 0.1)
+    y_noise[2] <- rnorm(1, mean = y_noise[2], sd = 100)
+
+    out <- run_oxygen_forecast(bc = boundary_noise, params = wq_parameters,
+                               ini = y_noise, times = times, ice = ice_on)
```

Comment:
I'm not sure how this function runs, but it would also be good to incorporate ensemble met drivers for forecast uncertainty

Reply:
Yes, so currently one can only run it with one ensemble met driver member, but I could change the code to accept a list of data frames with the ensemble met drivers. does that sound good?

Reply:
yup, that sounds great